### PR TITLE
[MASTER] EN-1587-pageTree-active-language

### PR DESCRIPTION
**Pull request by fedelocci**
add fields active languages

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: src/locales/it.js [hunk 1]
```diff
@@ -18,6 +18,281 @@ export default {
     'app.unpublish': 'Depubblica',
     'app.en': 'en',
     'app.it': 'it',
+    'app.aa': 'aa',
```
Comment (alessioscalici) [inline src/locales/it.js]: All these codes don't need a translation. Why are you including them in the translation files? Wouldn't a constant be enough?

File: src/locales/en.js [hunk 2]
```diff
@@ -18,6 +18,281 @@ export default {
     'app.unpublish': 'Unpublish',
     'app.en': 'en',
     'app.it': 'it',
+    'app.aa': 'aa',
```
Comment (alessioscalici) [inline src/locales/en.js]: All these codes don't need a translation. Why are you including them in the translation files? Wouldn't a constant be enough?

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**